### ISSUE TITLE
Can Night Light be made to Autostart?[Request] 

### ISSUE THREAD

**Issue by brent111** [BuddiesOfBudgie/budgie-desktop#492]
### Budgie version

10.8.2

### Use-cases

An autostart feature would help people who want to keep it warm and dimmer 24 hours a day. Setting nightlight to 5am to 5am does not work. Neither does 5 to 4:59. Can night light be started on boot?

### Proposal

Make the manual night light setting an autostart option if user wanted it to be. Or or users who prefer a warm screen all 24 hours of the day.

### References

![2019](https://github.com/BuddiesOfBudgie/budgie-desktop/assets/143135556/5c11f921-660e-420d-9210-b330da90a6ed)
 
Is this 2019 response (here at BoB) about the problem still relevant? Is there no way to autostart nightlight on boot?

**Comment by JoshStrobl** [comment]
Yes, it's still relevant. As I noted in that, it's a deficiency in GNOME Settings Daemon. It's not specific to Budgie Desktop. As you can see from the linked issues below, you can't set a custom schedule and even if you could, it's not respecting that at the moment (us northerners getting blasted with red in summer time when the sun goes down very late):

- https://gitlab.gnome.org/GNOME/gnome-settings-daemon/-/issues/419
- https://gitlab.gnome.org/GNOME/gnome-settings-daemon/-/issues/754
- https://gitlab.gnome.org/GNOME/gnome-settings-daemon/-/issues/764

I'm going to keep _this_ issue open and mark it for next major though so we know this is something we should address with our own daemon, once we have redshifting in magpie-wm and bigger items are out of the way.

**Comment by brent111** [comment]
Thank you for your time. Had overlooked Magpie.

**Comment by solomongrundy6** [comment]
There is a workaround for this that I've been using, from [here](https://askubuntu.com/questions/1087449/force-gnome-night-light-to-stay-on-and-never-turn-off).  It doesn't have the temporary bright flash on rollover like specifying a traditional time does.

- Open dconf-editor and navigate to the key `/org/gnome/settings-daemon/plugins/color/`
- Set `night-light-schedule-automatic` to false
- Set `night-light-schedule-from` to 0, and `night-light-schedule-to` to 24 (or any value higher than this)

**Comment by fossfreedom** [comment]
Is this still relevant under budgie 10.10? This whole area is now managed under different wayland based mechanisms.

**Comment by brent111** [comment]
I don't use X11 Budgie anymore and auto night light works in arch 10.10.2
